README.md

```diff
@@ -102,6 +122,13 @@ in lieu of command-line flags when using the above applications:
    reduces both latency and the number of Fleet API calls a client makes when
    reconnecting to a vehicle after restarting. This is particularly helpful
    when using `tesla-control`, which restarts on each invocation.
+ * `TESLA_TLS_CERT` specifies a TLS certificate file for the HTTP proxy.
```

Comment:
Does the Fleet Telemetry server also allow configuration through environment variables? This could become confusing.

Reply:
It's generally configured with `config.json` instead of flags/env vars.